clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTaskRepository.groovy

```diff
@@ -124,6 +126,8 @@ class JedisTaskRepository implements TaskRepository {
       pipe.expire(resultId, TASK_TTL)
       pipe.sync()
     }
+
+    log.debug("addResultObjects (resultId: ${resultId}, values: ${values})")
```

Comment:
An aside but we've see some situations where we appear to temporarily "lose" result objects (cc: @anotherchrisberry)